spec/controllers/skills_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe SkillsController do
+  describe "managing skills" do
+  
+    before do
+      Skill.create(skill: "Rspec")
+    end
+
+    context "adding skills" do
+      it "can add an skill" do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context "editing skills" do
+      it "can edit an skill" do
+        skills = Skill.find_by(skill: 'Rspec')
+        skills.skill = 'Business Development'
+        skills.save
+        response.should be_successful
+      end
+    end
+
+    context "destroy skills"  do
+      it "can destroy an skill" do
+        Skill.find_by(skill: 'Rspec').destroy
+        expect(@skill).to be_nil
```

Comment:
@skill doesn't appear to be defined anywhere so testing that @skill is nil will be true. On line 27 you should probably change it to `@skill = Skill.find_by(skill: 'Rspec').destroy` which will instantiate and destroy it.

Reply:
@seanmarcia  Ah yes you're right. I made this change as well as a few others for the skills_controller_spec. I also made the same changes to the Interest_controller_spec. It was written the same way.